src/main/java/com/nha/abdm/wrapper/hip/hrp/discover/DiscoveryService.java

```diff
@@ -26,14 +27,21 @@
 public class DiscoveryService implements DiscoveryInterface {
 
   @Autowired PatientRepo patientRepo;
-  @Autowired RequestManager requestManager;
+  private final RequestManager requestManager;
+  private final SessionManager sessionManager;
```

Comment:
I do not see the usage of `SessionManager` instance in this class, why is it needed?

Reply:
true, removed